lib/app/sequence_base.rb

```diff
@@ -743,7 +743,7 @@ def resolve_element_from_path(element, path)
       def get_value_for_search_param(element)
         case element
         when FHIR::Period
-          element.start || element.end
+          'gt' + element.start || 'lt' + element.end
```

Comment:
You'll need an actual condition here because `'gt' + element.start` is always truthy.